cmd/geth/chaincmd.go

```diff
@@ -311,7 +312,7 @@ func exportChain(ctx *cli.Context) error {
 	var err error
 	fp := ctx.Args().First()
 	if len(ctx.Args()) < 3 {
-		err = utils.ExportChain(chain, fp)
```

Comment:
Could you please move this into its own variable declaration, and maybe outside of the if?
```
exportReceipts := ctx.GlobalBool(utils.ExportReceiptsFlag.Name)
```